File: java/yapay-gateway-client/src/main/java/br/com/yapay/gateway/model/AirlineTripData.java
package br.com.yapay.gateway.model;

import com.google.gson.annotations.SerializedName;

public class AirlineTripData {

	@SerializedName("pnr")
	private String pnr;

	@SerializedName("titularCartaoEPassageiro")
	private Boolean cardholderIsPassenger;

	@SerializedName("dataEmissaoPassagem")
	private String ticketIssueDate;

	@SerializedName("codigoCompanhiaAerea")
	private String companyCode;

	@SerializedName("aeroportoPartida")
	private String departureAirport;

	@SerializedName("dataHoraPartida")
	private String departureDate;

	@SerializedName("aeroportoChegada")
	private String arrivalAirport;

	@SerializedName("dataHoraChegada")
	private String arrivalDate;

	public String getPnr() {
		return pnr;
	}

	public void setPnr(String pnr) {
		this.pnr = pnr;
	}

	public Boolean getCardholderIsPassenger() {
		return cardholderIsPassenger;
	}

	public void setCardholderIsPassenger(Boolean cardholderIsPassenger) {
		this.cardholderIsPassenger = cardholderIsPassenger;
	}

	public String getTicketIssueDate() {
		return ticketIssueDate;
	}

	public void setTicketIssueDate(String ticketIssueDate) {
		this.ticketIssueDate = ticketIssueDate;
	}

	public String getCompanyCode() {
		return companyCode;
	}

	public void setCompanyCode(String companyCode) {
		this.companyCode = companyCode;
	}

	public String getDepartureAirport() {
		return departureAirport;
	}

	public void setDepartureAirport(String departureAirport) {
		this.departureAirport = departureAirport;
	}

	public String getDepartureDate() {
		return departureDate;
	}

	public void setDepartureDate(String departureDate) {
		this.departureDate = departureDate;
	}

	public String getArrivalAirport() {
		return arrivalAirport;
	}

	public void setArrivalAirport(String arrivalAirport) {
		this.arrivalAirport = arrivalAirport;
	}

	public String getArrivalDate() {
		return arrivalDate;
	}

	public void setArrivalDate(String arrivalDate) {
		this.arrivalDate = arrivalDate;
	}

}

File: net/YapayGatewayLib/model/AirlineTripData.cs
using System;
using Newtonsoft.Json;

namespace YapayGatewayLib.Model
{
    public class AirlineTripData
    {
       [JsonProperty(PropertyName = "pnr")]
       public string pnr { get; set; }

       [JsonProperty(PropertyName = "titularCartaoEPassageiro")]
       public bool cardholderIsPassenger { get; set; }

       [JsonProperty(PropertyName = "dataEmissaoPassagem")]
       public string ticketIssueDate { get; set; }

       [JsonProperty(PropertyName = "codigoCompanhiaAerea")]
       public string companyCode { get; set; }

       [JsonProperty(PropertyName = "aeroportoPartida")]
       public string departureAirport { get; set; }

       [JsonProperty(PropertyName = "dataHoraPartida")]
       public string departureDate { get; set; }

       [JsonProperty(PropertyName = "aeroportoChegada")]
       public string arrivalAirport { get; set; }

       [JsonProperty(PropertyName = "dataHoraChegada")]
       public string arrivalDate { get; set; }
    }
}
